api/net/Areas/Subscriber/Models/NotificationModel.cs

```diff
@@ -0,0 +1,131 @@
+using System.Text.Json;
+using TNO.API.Models;
+using TNO.API.Models.Settings;
+using TNO.Entities;
+
+namespace TNO.API.Areas.Subscriber.Models;
+
+/// <summary>
+/// NotificationModel class, provides a model that represents an notification.
+/// </summary>
+public class NotificationModel : BaseTypeWithAuditColumnsModel<int>
+{
+    #region Properties
+    /// <summary>
+    /// get/set - The type of notification.
+    /// </summary>
+    public NotificationType NotificationType { get; set; }
+
+    /// <summary>
+    /// get/set - The foreign key to the template.
+    /// </summary>
+    public int TemplateId { get; set; }
+
+    /// <summary>
+    /// get/set - The Razor templates to generate the notification.
+    /// </summary>
+    public NotificationTemplateModel? Template { get; set; }
```

Comment:
This model doesn't need to return the template.  Only include what is needed.  Also review the properties below whether they are needed.  If the web app isn't going to use it, not sense serializing it.